StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewTests.swift

```diff
@@ -13,30 +13,96 @@ import WebKit
         let componentManager = EmbeddedComponentManager(apiClient: .init(publishableKey: "test"), fetchClientSecret: {
-            XCTFail("Client secret should not be retrieved in this test")
+            expectation.fulfill()
             return ""
         })
-        let webView = ConnectComponentWebView(componentManager: componentManager, componentType: .payouts)
+        let webView = ConnectComponentWebView(componentManager: componentManager, componentType: .payouts, loadContent: false)
         
-        XCTAssertEqual(webView.url, URL(string: "https://connect-js.stripe.com/v1.0/ios_webview.html#component=payouts&publicKey=test"))
+        try await webView.evaluateMessageWithReply(name: "fetchClientSecret",
+                                                   json: "{}")
+        
+        await fulfillment(of: [expectation], timeout: 1.0)
     }
     
     @MainActor
-    func testFetchClientSecret() async throws {
-        let expectation = XCTestExpectation(description: "Client secret is fetched")
+    func testFetchInitParams() async throws {
+        let expectation = self.expectation(description: "Message received")
+        let message = FetchInitParamsMessageHandler.Reply(locale: "fr-FR")
         let componentManager = EmbeddedComponentManager(apiClient: .init(publishableKey: "test"), fetchClientSecret: {
-            expectation.fulfill()
+            XCTFail("Client secret should not be retrieved in this test")
             return ""
         })
-        let webView = ConnectComponentWebView(componentManager: componentManager, componentType: .payouts)
+        let webView = ConnectComponentWebView(componentManager: componentManager, 
+                                              componentType: .payouts,
+                                              webLocale: Locale(identifier: "fr_FR"),
+                                              loadContent: false)
+       
         
-        try await webView.evaluateMessageWithReply(name: "fetchClientSecret",
+        try await webView.evaluateMessageWithReply(name: "fetchInitParams",
                                                    json: "{}",
-                                                   postReply: false)
+                                                   verifyResult: { result in
+            guard let json = try? JSONEncoder.connectEncoder.encode(message) else {
+                XCTFail("Failed to encode message \(message)")
+                return
+            }
+            XCTAssertEqual(String(data:result, encoding: .utf8), String(data:json, encoding: .utf8))
+            expectation.fulfill()
+        })
+        await fulfillment(of: [expectation], timeout: 0.2)
+    }
+    
+    @MainActor
+    func testUpdateAppearance() async throws {
+        let componentManager = EmbeddedComponentManager(apiClient: .init(publishableKey: "test"), fetchClientSecret: {
+            XCTFail("Client secret should not be retrieved in this test")
+            return ""
+        })
```

Comment:
optional nit: it looks like this is used a lot in almost every test. Should it get DRYed up into a helper?